tests/Phinx/Db/Adapter/SQLiteAdapterTest.php

```diff
@@ -41,6 +41,13 @@ public function testConnection()
         $this->assertTrue($this->adapter->getConnection() instanceof \PDO);
     }
 
+    public function testBeginTransaction()
+    {
+        $this->adapter->getConnection()
+            ->setAttribute(\PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION);
+        $this->adapter->beginTransaction();
```

Comment:
Added `assertTrue(true)` assertion as we are only testing the validity of the query.